coalib/settings/ConfigurationGathering.py

```diff
@@ -135,27 +135,30 @@ def warn_nonexistent_targets(targets, sections, log_printer):
 
```

Comment:
About the shortlog, I think the current one is more descriptive for the change and it doesn't exceed the char limit.